conan/tools/cmake/toolchain/blocks.py

```diff
@@ -357,15 +393,30 @@ def _apple_sdk_name(self):
 
     def context(self):
         os_ = self._conanfile.settings.get_safe("os")
-        if os_ not in ['Macos', 'iOS', 'watchOS', 'tvOS']:
+        if not is_apple_os(os_):
             return None
 
         arch = self._conanfile.settings.get_safe("arch")
         host_architecture = to_apple_arch(arch)
         host_os_version = self._conanfile.settings.get_safe("os.version")
         host_sdk_name = self._apple_sdk_name()
+        is_debug = self._conanfile.settings.get_safe('build_type') == "Debug"
 
-        ctxt_toolchain = {}
+        # Reading some configurations to enable or disable some Xcode toolchain flags and variables
+        # Issue related: https://github.com/conan-io/conan/issues/9448
+        # Based on https://github.com/leetal/ios-cmake repository
+        enable_bitcode = self._conanfile.conf.get("tools.apple:enable_bitcode",
```

Comment:
Are we sure the default for watchOS and tvOS should be ``True``? If that is the default, why the CMake or compiler is not setting them and Conan should set it?

Reply:
no, it shouldn't